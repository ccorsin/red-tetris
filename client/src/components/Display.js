import React from 'react';
import { StyledDisplay, StyledDisplayCase } from './styles/StyledDisplay';

const Display = ({ gameOver, text, number }) => (
  <StyledDisplay>
    {
      gameOver ? (
        <StyledDisplayCase>
          <p>{text}</p>
        </StyledDisplayCase>

      ) : (
        <StyledDisplayCase>
          <p>{text}</p>
          <p>{number}</p>
        </StyledDisplayCase>
      )
    }
    </StyledDisplay >
);

export default Display;
