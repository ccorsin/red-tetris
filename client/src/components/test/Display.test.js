import React from 'react'
import { shallow } from 'enzyme';
import Display from '../Display';
import toJson from "enzyme-to-json"

describe('<Display/> Component', () => {
  it('rendering correctly Display', () => {
    let wrapper = shallow(
      <Display text='text' number='42'/>
    )
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})