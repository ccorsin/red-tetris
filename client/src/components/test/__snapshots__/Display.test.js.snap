// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Display/> Component rendering correctly Display 1`] = `
<styled.div>
  <styled.div>
    <p>
      text
    </p>
    <p>
      42
    </p>
  </styled.div>
</styled.div>
`;
